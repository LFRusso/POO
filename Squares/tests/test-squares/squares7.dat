Square0 2.91447 -2.1336 2.22428
Square1 -0.645322 -0.966308 3.98887
Square2 0.674667 -1.25548 0.359119
Square3 -0.10489 2.88382 3.86307
Square4 -1.26046 0.25643 4.91036
Square5 -2.92123 2.96533 1.07639
Square6 -1.56805 2.06804 1.23495
Square7 0.355836 -1.6024 4.76378
Square8 1.07411 1.37519 3.37805
Square9 1.01827 0.180733 0.634541
Square10 -0.0892513 -2.21085 1.46599
Square11 -0.64429 2.05944 3.16856
Square12 2.18925 -1.91079 2.10211
Square13 -0.949425 0.673795 4.87604
Square14 -0.737275 -0.956716 2.42022
