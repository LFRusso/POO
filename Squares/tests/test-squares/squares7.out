Square0 intercepts Square1 with area 0.45353
Square0 intercepts Square7 with area 3.73349
Square0 intercepts Square12 with area 2.7558
Square1 intercepts Square2 with area 0.0251193
Square1 intercepts Square3 with area 0.478443
Square1 intercepts Square4 with area 11.0337
Square1 intercepts Square6 with area 0.298023
Square1 intercepts Square7 with area 11.9176
Square1 intercepts Square8 with area 3.73861
Square1 intercepts Square9 with area 0.402642
Square1 intercepts Square10 with area 0.324641
Square1 intercepts Square11 with area 3.05171
Square1 intercepts Square12 with area 1.33625
Square1 intercepts Square13 with area 9.36893
Square1 intercepts Square14 with area 5.63492
Square2 intercepts Square7 with area 0.128966
Square2 intercepts Square10 with area 0.128966
Square2 intercepts Square14 with area 0.0216746
Square3 intercepts Square4 with area 8.57207
Square3 intercepts Square7 with area 0.944355
Square3 intercepts Square8 with area 5.01765
Square3 intercepts Square11 with area 6.16322
Square3 intercepts Square13 with area 10.299
Square4 intercepts Square6 with area 1.14524
Square4 intercepts Square7 with area 9.56909
Square4 intercepts Square8 with area 8.70115
Square4 intercepts Square9 with area 0.354609
Square4 intercepts Square11 with area 9.84582
Square4 intercepts Square13 with area 20.6647
Square4 intercepts Square14 with area 2.92138
Square6 intercepts Square11 with area 0.384304
Square6 intercepts Square13 with area 0.761131
Square7 intercepts Square8 with area 6.03384
Square7 intercepts Square9 with area 0.402642
Square7 intercepts Square10 with area 0.875465
Square7 intercepts Square11 with area 2.38948
Square7 intercepts Square12 with area 3.7706
Square7 intercepts Square13 with area 8.88262
Square7 intercepts Square14 with area 3.2119
Square8 intercepts Square11 with area 3.90644
Square8 intercepts Square13 with area 9.6359
Square8 intercepts Square14 with area 0.0537687
Square9 intercepts Square13 with area 0.0897742
Square9 intercepts Square14 with area 0.402642
Square10 intercepts Square14 with area 0.310579
Square11 intercepts Square13 with area 10.0398
Square13 intercepts Square14 with area 1.91127
